resources/migrate-to-swift.md

```diff
@@ -47,10 +47,34 @@ If you are using Cocoapods to manage your Adobe Experience Platform Mobile SDK d
   pod 'AEPUserProfile'
 ```
 
-Save the `Podfile` and run `pod repo update` to update your local Cocoapods repository.
+Save the `Podfile` and run `pod repo update` to update your local CocoaPods repository.
 
 Once the previous command is complete, run `pod install` or `pod update` to update the application dependencies.
 
+### SPM
```

Comment:
Since this is a header, I'd go with the full title, and go with "Swift Package Manager".